week-3-pratise/exercise1.js

```diff
@@ -0,0 +1,16 @@
+/* Given is an array find the element with characters more than 6 */
+
+var food = ["Pizza", "Burger", "Fries", "Kebab", "Chicken Wings"];
+
+var findLongestFoodName = food.filter(function(food) {
```

Comment:
Is this correct? You might need to ask the person who created the exercise!

It says in the description to *find the element*, so I think they mean you should only find 1 answer. But your solution could find multiple answers it's just chance that there's only 1 element in the array that matches (if you added "Lasagne" to the array would it still work?)

But I might be reading the question wrong. The *intention* is not clear